releasenotes/notes/0.25/faster-parameter-rebind-3c799e74456469d9.yaml

```
@@ -1,13 +1,13 @@
 ---
-features:
+features_circuits:
   - |
     The performance of :meth:`.QuantumCircuit.assign_parameters` and :meth:`~.QuantumCircuit.bind_parameters`
     has significantly increased for large circuits with structures typical of applications uses.
     This includes most circuits based on the :class:`.NLocal` structure, such as
     :class:`.EfficientSU2`.  See `#10282 <https://github.com/Qiskit/qiskit-terra/issues/10282>`__ for more
     detail.
   - |
-    The method :meth:`.QuantumCircuit.assign_parameters` has gained two new keywords arguments: ``flat_input``
+    The method :meth:`.QuantumCircuit.assign_parameters` has gained two new keywords arguments:: ``flat_input``
```

This was correct before, two `::` means a code block.
```suggestion
    The method :meth:`.QuantumCircuit.assign_parameters` has gained two new keywords arguments: ``flat_input``
```